default/web_tt2/report_abuse.tt2

```diff
@@ -0,0 +1,5 @@
+<!-- report_abuse.tt2 -->
+[% IF conf.show_report_abuse %]
+    [% | mailto("${conf.listmaster_email}@${domain}",{subject=>"Sympa abuse report: ${list}"}) %][%|loc%]Report abuse[%END%][%END%]
+[% END %]
+<!-- end report_abuse.tt2 -->
```

Comment:
`[% IF ... END %]` is redundant: One in `list_menu.tt2` is enough.

Reply:
I first wanted to put the if in `report_abuse.tt2` then noticed than I needed one in `list_menu.tt2` (because of the `<li>`) but forgot to remove it here.

Fixed now.